arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -2486,9 +2493,18 @@ void IResearchAnalyzerFeature::prepare() {
   if (!isEnabled()) {
     return;
   }
-
-  // load all known analyzers
-  ::iresearch::analysis::analyzers::init();
+  ::iresearch::analysis::delimited_token_stream::init();
+  ::iresearch::analysis::collation_token_stream::init();
+  ::iresearch::analysis::ngram_token_stream_base::init();
+  ::iresearch::analysis::normalizing_token_stream::init();
+  ::iresearch::analysis::stemming_token_stream::init();
+  ::iresearch::analysis::text_token_stream::init();
+  ::iresearch::analysis::token_stopwords_stream::init();
+  ::iresearch::analysis::pipeline_token_stream::init();
+  ::iresearch::analysis::segmentation_token_stream::init();
+#ifdef USE_ENTERPRISE
+  initAnalyzersWithModel();
```

Comment:
NIT: Maybe less "concrete" name. Something like "initEnterpriseAnalyzers". As there may be others non-model related analyzers.